hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/records/TxnAwareRecordsHistorianTest.java

```diff
@@ -597,4 +607,70 @@ private void verifyBuilderUse(
         verify(builder).setTxnId(expectedId);
         verify(builder).setTxnHash(expectedHash);
     }
+
+    private ExpirableTxnRecord.Builder contractCreateAndCallResult() {
```

Comment:
No usages of this new method?  What are the plans for this?

If you're not going to use it right away consider moving it (and all of the supporting static variables) to another "test helper" file.

And BTW if all those supporting static variables are just for internal use of this specific method then I suggest moving them right into the method providing them with a tighter scope.  It's not important that they're declared `static` even if they effectively are.